src/main/kotlin/next/step/blackjack/domain/GameBoard.kt

```diff
@@ -0,0 +1,32 @@
+package next.step.blackjack.domain
+
+import next.step.blackjack.domain.player.Player
+
+data class GameBoard(val gameCards: GameCards, val players: Set<Player>) {
```

Comment:
```suggestion
data class GameBoard(val gameCards: GameCards, val players: Players) {
```

Players도 일급 컬렉션으로 만들어서 GameBoard와 Players 로직을 분리해보죠!